src/components/chip/chip.e2e.ts

```diff
@@ -0,0 +1,57 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('limel-chip', () => {
+    let page;
+
+    beforeEach(async () => {
+        page = await newE2EPage();
+    });
+
+    it('should render with the correct text', async () => {
+        await page.setContent('<limel-chip text="Test chip"></limel-chip>');
+        await page.waitForChanges();
+
+        const element = await page.find('limel-chip >>> .chip');
+        expect(element.textContent).toEqual('Test chip');
+    });
+
+    it('should render as a link when the link prop is provided', async () => {
+        await page.setContent(
+            '<limel-chip text="Test chip" link="{ href: \'https://example.com\' }"></limel-chip>',
+        );
+
+        const element = await page.find('limel-chip');
+        element.setProperty('link', { href: 'https://example.com' });
+
+        await page.waitForChanges();
+
+        expect(element)
+            .toEqualHtml(`<limel-chip class="hydrated" link="{ href: 'https://example.com' }" text="Test chip">
+        <mock:shadow-root>
+          <a class="chip" href="https://example.com" tabindex="0">
+            <span class="text">
+              Test chip
+            </span>
+          </a>
+        </mock:shadow-root>
+      </limel-chip>`);
+    });
+
+    it('should render with a badge when the badge prop is provided', async () => {
```

Comment:
Regarding `it('should render …`, we typically just write `it('renders …`.

Just an FYI, I've already fixed it locally and will be pushing a fixup.